be/src/main/java/kr/codesquad/sidedish/response/CommonCode.java

```diff
@@ -9,6 +9,7 @@
 public enum CommonCode {
 
 	SUCCESS(HttpStatus.OK, "성공"),
+	SUCCESS_NO_CONTENT(HttpStatus.OK, "요청에 성공했으나 응답할 데이터가 없습니다."),
```

Comment:
```suggestion
	SUCCESS_NO_CONTENT(HttpStatus.OK, HttpStatus.NO_CONTENT.getReasonPhrase()),
```

`nit`
저라면 걍 이렇게 퉁쳤을 것 같기도 해요.